src/_pytest/mark/structures.py

```diff
@@ -82,39 +81,23 @@ def param(cls, *values, **kw):
         return cls(values, marks, id_)
 
     @classmethod
-    def extract_from(cls, parameterset, belonging_definition, legacy_force_tuple=False):
+    def extract_from(cls, parameterset, force_tuple=False):
```

Comment:
that parameter name shouldnt be changed
also taking away belonging definition is a breaking change

Reply:
actually right, that one is a internal one, we should make it a normal function, so its not accidentially exposed on instances

Reply:
That's a good point, but I think that's not even necessary because one can't even access a `ParameterSet` instance, even from hooks (I think).

Reply:
@nicoddemus its exposed as return value of pytest.param